requirements.txt

```diff
@@ -2,16 +2,5 @@
 django>=1.8,<1.9
 django-model-utils==2.3.1
```

Comment:
These requirements do not play into the edx-platform install. When this package is installed into the edx-platform virtualenv, pip will only be looking at the requirements listed in the `install_requires` list in setup.py.  That said, the requirements/versions here should match what is in setup.py and vice versa. I will update to remove this discrepancy.